op.yml

```diff
@@ -1,3 +1,41 @@
+---
 name: github.com/opspec-pkgs/docker.push.localimage
-description: push docker image that is stored on host machine
+description:
+  Builds image that stores on host machines docker image store.
```

Comment:
was this meant to be push?

Reply:
Yikes! Yes. I'll fix and re-submit. Thanks for the catch!